### PR TITLE
Add sentry

### PR DESCRIPTION
**Pull request by masoncm97**

### CODE REVIEW

**Comment by one1zero1one** [comment]
Hey @masoncm97 nice to meet you. I've been helping @genekogan with infra in the past year. 
What do you think about https://develop.sentry.dev/sdk/performance/opentelemetry/  ? Using opentelemetry instead of the "vendor lock" sentry SDK would save  so much time in the future with refactoring - and would enable in addition to sentry shipping spans to one of the o11y backed providers (such as honeycomb.io, lightstep, etc) - or locally hosted ones (signoz, elastic) - giving us the power to ask arbitrary questions, not only in the error space, but also in the actual logic at scale. Thanks for sharing your thoughts on this.

**Comment by masoncm97** [comment]
Hey Daniel, it's nice to meet you! Gene and I discussed using OpenTelemetry, but erred on the side of Sentry.io in the initial phase primarily due to it's out-of-the-box usage. If you think Open Telemetry is a better fit then I'm happy to look into it, however I must admit that my knowledge of the observability realm is somewhat limited- which is part of the reason Gene and I opted for Sentry.io in the first place. 

A quick question, if you don't mind: 

Do you think the following library would be useful when setting up the instrumentation from Eden API? I'm trying to conceptualize how I can refactor this PR to achieve the same workflow but using OT rather than Sentry for instrumentation. 

https://www.npmjs.com/package/@opentelemetry/instrumentation-fastify

**Comment by one1zero1one** [comment]
@masoncm97  I'm not familiar with this iteration of the api as of yet to test it myself - and there were some changes in the PR that threw me of that seemed unrelated to sentry (my typescript foo is not that strong). In essence Sentry is absolutely better than what is in place now, live logs on multiple instances that need to be consulted by hand, so in that respect if this works as intended ship it 🚀  no need to refactor right now 😄 

Regarding otel, autoinstrumentation works kinda in the same way, e.g. https://www.npmjs.com/package/@opentelemetry/auto-instrumentations-node - basically it's a wrapper that will send a bunch of spans automatically when enabled on a specific thing (like fastify, or mongodb). The auto instrumentation is a good start, but adding attributes to spans is what makes the solution truly observable, but there's a bit more work there to get things rolling, setting up otel pipelines - and getting confortable with a backend that can handle metrics/logs/traces (honeycomb.io, lightstep, etc)

**Review by one1zero1one — APPROVED** [review]
lgtm

File: src/index.ts [hunk 1]
```diff
@@ -1,10 +1,16 @@
 import createServer from './server';
+import * as Sentry from '@sentry/node';
 
 process.on('unhandledRejection', (err) => {
   console.error(err);
   process.exit(1);
 });
 
+Sentry.init({
+  dsn: process.env.SENTRY_DSN,
```
[the remark this answers is not in the document]
Comment (genekogan) [reply src/index.ts:10]: it is running in mars namespace. i'm ready to merge this, provided the DSN value. i think it needs to be kept secret though, so i will add it when i get it.

**Pull request merged**